From 507c394cfcf4edffc5e4450c5d737e545c26b857 Mon Sep 17 00:00:00 2001
From: Daniel Engberg <daniel.engberg.lists@pyret.net>
Date: Sat, 12 Dec 2020 18:56:38 +0100
Subject: [PATCH] p11-kit/lists.c: Add stdint.h to fix compilation

Add stdint.h otherwise compilation fails on FreeBSD 13-CURRENT with "use of undeclared identifier 'SIZE_MAX'"

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>
---
 p11-kit/lists.c | 1 +
 1 file changed, 1 insertion(+)

--- a/p11-kit/lists.c
+++ b/p11-kit/lists.c
@@ -39,6 +39,7 @@
 
 #include <assert.h>
 #include <ctype.h>
+#include <stdint.h>
 #include <string.h>
 #include <stdio.h>
 #include <stdlib.h>
